vita3k/gxm/include/gxm/types.h

```diff
@@ -1336,14 +1336,24 @@ using SceGxmVertexOutputTexCoordInfos = std::array<uint8_t, 10>;
 
 #pragma pack(push, 1)
 struct SceGxmProgramVertexVaryings {
-    std::uint8_t unk0[8];
-    std::uint8_t fragment_output_start; // this might be wrong
-    std::uint8_t unk1;
-    std::uint8_t output_param_type;
-    std::uint8_t output_comp_count;
-
-    std::uint16_t varyings_count;
-    std::uint16_t pad0; // padding maybe
+    union {
```

Comment:
Is this really union? Didn't you intend to replace unk0 with attrib_pa_regs and untyped_pa_regs?

Reply:
Something you might want to consider is SceGxmProgramVertexVaryings of fragment program is the input vertex attributes. Maybe it's just better to change the name of this struct. Although I'm not sure about unk0 of vertex program.

Reply:
Oh I just noticed that you only marked some part of struct as union. varyings_count should be outside of union. It's defined in fragment program.

Reply:
Alright top one is for vertex and bottom one is for fragments I see. Ignore all my comments lol.

Reply:
👍

Reply:
Ok, I did that and also named unk18.